src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +193,20 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface Trigger{
+    val id: String
+    val type: String
+
+    fun fetchProperties(): List<Parameter>
+}
+
+interface ArtifactDependency{
+    val id: String
```

Comment:
What is id here? Is it id of dependency? Or of the artifact? Or of build configuration it refers to?